src/main/java/uk/gov/hmcts/reform/coh/controller/DeadlineController.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.reform.coh.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.coh.domain.OnlineHearing;
+import uk.gov.hmcts.reform.coh.service.OnlineHearingService;
+import uk.gov.hmcts.reform.coh.service.QuestionService;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/continuous-online-hearings/{onlineHearingId}")
```

Comment:
/continuous-online-hearings/{onlineHearingId}/questions because the extension is for questions

Reply:
Was worried about overlapping `/continuous-online-hearings/{onlineHearingId}/questions/{questionId}`, where `deadline-extensions` could be mistaken as question ID.

Reply:
Something like /continuous-online-hearings/{onlineHearingId}/questions/deadline-extensions should be fine, no?